app/web/src/i18n/resources.ts

```diff
@@ -0,0 +1,23 @@
+import auth from "features/auth/locales/en.json";
+import communities from "features/communities/locales/en.json";
+import connections from "features/connections/locales/en.json";
+import dashboard from "features/dashboard/locales/en.json";
+import donations from "features/donations/locales/en.json";
+import messages from "features/messages/locales/en.json";
+import profile from "features/profile/locales/en.json";
+import search from "features/search/locales/en.json";
+import global from "resources/locales/en.json";
+
+const resources = {
+  auth,
+  communities,
+  connections,
+  dashboard,
+  donations,
+  messages,
+  profile,
+  search,
+  global,
+} as const;
+
```

Comment:
Since this is needed for the TypeScript types (in `src/i18n/react-i18next-types.d.ts` and the test setup of `i18n`), I thought it makes sense to put it in this separate file so we only have to update this in one place, then the types and the test i18n instance will update automatically.